app/views/cases/search_filters/_final_deadline.html.slim

```diff
@@ -0,0 +1,51 @@
+= form_for @query, url: search_cases_path, method: :get do |f|
+  = hidden_field_tag('search_query[filter_type]', 'final_deadline')
+  = hidden_field_tag('search_query[parent_id]', @parent_id)
+  fieldset
+    legend
+      .bold-medium
+        = "Filter by final deadline dates"
+
+    .grid-row
+      .column-full
+        .js-quick-links
+          span#quick-link-label.bold-small
+             = "Due"
+          ul.quick-link-options
+            li.quick-link-option--first
+              - today = {day: @query.available_deadlines[:today].strftime("%d"), month: @query.available_deadlines[:today].strftime("%m"), year: @query.available_deadlines[:today].strftime("%Y")}.to_json
+              a { href="#"
+                  aria-describeby="quick-link-label"
+                  data-external-deadline-from=today
+                  data-external-deadline-to=today
+                }
+                = "Today"
+            li.quick-link-option
+              - days_3 = {day: @query.available_deadlines[:three_days].strftime("%d"), month: @query.available_deadlines[:three_days].strftime("%m"), year: @query.available_deadlines[:three_days].strftime("%Y")}.to_json
+              a { href="#"
+                  aria-describeby="quick-link-label"
+                  data-external-deadline-from=today
+                  data-external-deadline-to=days_3
+                }
+                = "In the next 3 days"
+            li.quick-link-option
+              - days_10 = {day: @query.available_deadlines[:ten_days].strftime("%d"), month: @query.available_deadlines[:ten_days].strftime("%m"), year: @query.available_deadlines[:ten_days].strftime("%Y")}.to_json
+              a { href="#"
+                  aria-describeby="quick-link-label"
+                  data-external-deadline-from=today
+                  data-external-deadline-to=days_10
+                  }
+                = "In the next 10 days"
+
+    hr
+      .form-group
```

Comment:
We can remove this, v3 of the date field gems now includes it

Reply:
We agreed to ignore this for the time being there does seem to be a bug with the govuk date gem where its displaying the fields along side each other.